### PR TITLE
Local Fast Food Added

### PR DESCRIPTION
**Pull request by shahnehal0407**
Added a new template.
This template was created for local fast food shop. It consist of food gallery and also costumer can read review and all add their review.

Fixes:  #800

# Checklist:

- [x] I have taken help from some online resourses

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

Taken help from YT but video link is not available.
preview

### CODE REVIEW

**Comment by shahnehal0407** [comment]
> Added a new template. This template was created for local fast food shop. It consist of food gallery and also costumer can read review and all add their review.
> 
> Fixes: #825
> 
> # Checklist:
> * [x]  I have made this from my own
> * [x]  My code follows the style guidelines of this project
> * [x]  I have performed a self-review of my own code
> * [x]  I have commented my code, particularly in hard-to-understand areas
> * [x]  I have made corresponding changes to the documentation
> * [x]  My changes generate no new warnings
> 
> preview

@DEEPIKA-SUK please review it

File: Local Fast Food/index.html [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+<head>
+    <link rel="stylesheet" type="text/css" href="vendors/css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="vendors/css/Grid.css">
+    <link rel="stylesheet" type="text/css" href="vendors/css/animate.css">
+
+
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Food Fast</title>
```
Comment (VidhiBhatt01) [inline Local Fast Food/index.html:13]: Fast Food and not Food Fast

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
1. Link issue number not previous PR number.
2. Fix these
image
3. Are you sure you made this on your own!?👀

**Comment by VidhiBhatt01** [comment]
https://deploy-preview-849--cool-front-end-templates.netlify.app/local%20fast%20food/

**Review by DEEPIKA-SUK — CHANGES_REQUESTED** [review]
@shahnehal0407 fix this
Screenshot 2022-05-21 094903
.

**Review by DEEPIKA-SUK — CHANGES_REQUESTED** [review]
@shahnehal0407 update the title in html file.

**Review by VidhiBhatt01 — APPROVED** [review]

**Pull request merged**